userpanel/modules/finances/templates/balance.html

```diff
@@ -2,7 +2,7 @@
 {box title="Balance"}
 <center>
 <h1>{if $userinfo.balance < 0}
-	<div style="color: red; white-space: nowrap;"><b>{$userinfo.balance|money_format}</b></div>
+	<div style="color: red; white-space: nowrap;"><strong>{$userinfo.balance|money_format}</strong></div>
```

Comment:
tu od razu proponuję zmienić klasę na bootstrapową `text-nowrap`.

Reply:
to jest stara skórka - działa tutaj bootstrap?

Reply:
dla starej skórki nie zmieniamy oczywiście na klasy bootstrap - moja pomyłka